### PR TITLE
Bug fixes on package, request for re-upload

### PR DESCRIPTION
**Pull request by ganiyuolalekan**
I just fixed some bugs during the course testing the package. Two things were changed

1. Added `Colorama` to requires.txt
2. Fixed a bug on DQA for duplicate assessment

A reupload to PyPI is required to effect these changes

### CODE REVIEW

**Review by AdesholaAfolabi — APPROVED** [review]

File: slik_wrangler/preprocessing.py [hunk 1]
```diff
@@ -436,7 +436,7 @@ def drop_duplicate(dataframe=None,columns=None,method='rows',display_inline=True
     elif method == 'columns':
         if columns is None:
             raise ValueError("columns: A list/string is expected as part of the inputs to columns, got 'None'")
-        dataframe = dataframe.drop_duplicates(subset=columns)
+        dataframe = dataframe.T.drop_duplicates().T
```
Comment (AdesholaAfolabi) [inline slik_wrangler/preprocessing.py:439]: What does .T do here?

File: slik_wrangler/dqa.py [hunk 2]
```diff
@@ -93,12 +94,12 @@ def check(col1, col2):
     
     if len(duplicated_rows):
         log(
-            f"Dataframe contains duplicate rows that you should address. \n\ncolumns={list(dupulicated_rows.index)}\n", 
+            f"Dataframe contains duplicate rows that you should address. \n\ncolumns={list(duplicated_rows.index)}\n",
```
Comment (AdesholaAfolabi) [inline slik_wrangler/dqa.py:97]: I remember I fixed this somewhere.

Comment (AdesholaAfolabi) [reply slik_wrangler/dqa.py:97]: Did you pull the latest changes before making a commit?

File: slik_wrangler/preprocessing.py (hunk 1 above)
Comment (ganiyuolalekan) [reply slik_wrangler/preprocessing.py:439]: It translate the dataset, so the columns takes the position of the rows and the duplicate columns are then dropped, and finally it is translated to its original state.

Comment (AdesholaAfolabi) [reply slik_wrangler/preprocessing.py:439]: Yeah, it's the transpose function. Wondering why you had to do it here? Did you test it?

File: slik_wrangler/dqa.py (hunk 2 above)
Comment (ganiyuolalekan) [reply slik_wrangler/dqa.py:97]: Nope I didn't

Comment (AdesholaAfolabi) [reply slik_wrangler/dqa.py:97]: lol always pull changes first before making a commit.

**Pull request closed without merging**